### PR TITLE
Remove packer dependency and use go-embed

### PR DESCRIPTION
**Pull request by ph**
This is a first step to remove packer module, the remaining will be done
when we move to v2.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: internal/pkg/release/pgp.go [hunk 1]
```diff
@@ -2,23 +2,14 @@
...
-}
-
 // PGP return pgpbytes and a flag describing whether or not no pgp is valid.
 func PGP() (bool, []byte) {
 	return allowEmptyPgp == "true", pgpBytes
```
Comment (ph) [inline internal/pkg/release/pgp.go:15]: I need to check how this is used this should not be a string at all.
will do in a followup

**Comment by ph** [comment]
@blakerouse if we merge this you should be able to remove the package module in the v2 refactor.

**Review by michel-laterman — COMMENTED** [review]
Are we going to try to remove the other instances of the packer dependency from the codebase?

**Comment by ph** [comment]
@michel-laterman The complete removal will go with v2 change.

File: internal/pkg/release/pgp.go (hunk 1 above)
Comment (michalpristas) [reply internal/pkg/release/pgp.go:15]: this is used during verification process of downloaded artifacts.
allowEmptyPgp is a string bacause it's a variable set during build time and there is/was restriction to use string

**Comment by ph** [comment]
@mergifyio update

**Comment by ph** [comment]
test

**Comment by ph** [comment]
test

**Comment by ph** [comment]
This should wait for v2.

**Pull request closed without merging**